app/helpers/articles_helper.rb

```diff
@@ -1,5 +1,16 @@
 module ArticlesHelper
 
+  def article_tag(article, &block)
+    klasses = ["h-entry"]
+    klasses << "article-with-no-header-image" if article.image.blank?
+
+    # Data attributes are used to determine how the article should be polled for updates
+    data = {id: article.id, published_at: Time.now.to_i}
```

Comment:
I'll be checking for `published_at` on or after this time. Not the original time of the published post. If the root post was published 24 hours ago, and 10 new posts since, those 10 new posts are already on the page, I don't want to poll for those 10 new posts. Only posts that have been published after `Time.now`.